src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -815,7 +822,8 @@ public void writeEntityToNBT(final NBTTagCompound compound)
             compound.setInteger(TAG_CITIZEN, citizenData.getId());
         }
 
-        inventory.writeToNBT(compound);
+        NBTTagCompound inv = inventory.serializeNBT();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'inv' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)